content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -7,31 +7,36 @@ slug: dockershim-faq
 aliases: [ '/dockershim' ]
 ---
 
-**This is an update to the original [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article,
-published in late 2020.**
+**This supersedes the original
+[Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article,
+published in late 2020. The article includes updates from the v1.24
+release of Kubernetes.**
 
 This document goes over some frequently asked questions regarding the
-deprecation and removal of _dockershim_, that was
+removal of _dockershim_ from Kubernetes. That removal was originally
```

Comment:
nit

```suggestion
removal of _dockershim_ from Kubernetes. The removal was originally
```

Reply:
Not sure which is more idiomatic, but happy to revise it.